opencti-platform/opencti-front/src/private/components/data/drawers/PromoteDrawer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import Alert from '@mui/material/Alert';
+import Button from '@mui/material/Button';
+import Drawer from '@components/common/drawer/Drawer';
+import { useFormatter } from '../../../../components/i18n';
+import type { Theme } from '../../../../components/Theme';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  buttons: {
+    marginTop: 20,
+    textAlign: 'right',
+  },
+  button: {
+    marginLeft: theme.spacing(2),
+  },
+  container: {
+    padding: '10px 20px 20px 20px',
+  },
+}));
```

Comment:
`makeStyles` is deprecated, use properties `sx` and `style` directly in JSX instead

Reply:
You can have a look at `WidgetTimeline.tsx` if you need an example